clinica/pydra/t1_volume/register_dartel/cli.py

```diff
@@ -24,26 +26,36 @@ def cli(
     group_label: str,
     dartel_tissues: List[int] = (1, 2, 3),
 ) -> None:
-    """Inter-subject registration using Dartel (using an existing Dartel template).
+    """Inter-subject registration using an existing Dartel template (Pydra engine).
 
     Parameters
     ----------
+    bids_directory : str
+        Path to the input BIDS directory.
+
+    caps_directory : str
+        Path to the CAPS directory.
 
     group_label : str
-        User-defined identifier to target a specific group of subjects. For this pipeline, it is associated to the
-        DARTEL template that you had created when running the t1-volume pipeline.
+        User-defined identifier to target a specific group of subjects.
+        For this pipeline, it is associated to the DARTEL template that was created
+        when running the t1-volume pipeline.
+
+    dartel_tissues : List[int], optional
```

Comment:
FYI, `numpydoc` recommends to use more litteral type names like `list of int` rather than raw Python annotations like `List[int]`.